### PR TITLE
Rules Page of my quiz

### PR DESCRIPTION
**Pull request by utkarshducoder**
please review my rules page

live link: https://quizzyfuy.netlify.app/quiz%20html/rules.html

Following are the features of this page:

- nav: this feature is made to add headings.
- section: this contains the main documentation part of the question like heading and the rules in the form of a link.
-footer: this feature is added to add the link for all of my social websites.

### CODE REVIEW

File: rules/rules.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro&display=swap');
+:root{
+    --primary-color: black;
+    --secondary-color:#fff;
+    box-sizing: border-box;
+
```
Comment (khushijohri2001) [inline rules/rules.css]: remove extra space

File: rules/rules.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    
+}
+
+li{
+    padding: 1.2rem;
+
+
+}
+footer li:hover{
+    color: red;
```
Comment (khushijohri2001) [inline rules/rules.css]: use var here

File: rules/rules.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    cursor: pointer;
+}
+.foot-info{
+    color: #fff;
```
Comment (khushijohri2001) [inline rules/rules.css]: use var here

**Review by khushijohri2001 — APPROVED** [review]

File: rules/rules.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (i-am-a-codegirl) [inline rules/rules.html]: add title rest is good to merge

**Review by i-am-a-codegirl — APPROVED** [review]

**Comment by khushijohri2001** [comment]
Good work but need to write code more cleaner without extra spaces and proper indentation. I would suggest you to use prettier tool or beautify, run it once in your code editor to make your code look well aligned and neat.

**Pull request merged**